diff --git a/apps/demo/demo_ble/board/wl82/board.c b/apps/demo/demo_ble/board/wl82/board.c
--- a/apps/demo/demo_ble/board/wl82/board.c
+++ b/apps/demo/demo_ble/board/wl82/board.c
@@ -9,10 +9,10 @@
 
-UART2_PLATFORM_DATA_BEGIN(uart2_data)
-	.baudrate = 1000000,
+UART1_PLATFORM_DATA_BEGIN(uart1_data)
+	.baudrate = 460800,
 	.port = PORT_REMAP,
 	.output_channel = OUTPUT_CHANNEL0,
-	.tx_pin = IO_PORTC_03,
-	.rx_pin = -1,
+	.tx_pin = IO_PORTB_03,
+	.rx_pin = IO_PORTA_06,
 	.max_continue_recv_cnt = 1024,
 	.idle_sys_clk_cnt = 500000,
 	.clk_src = PLL_48M,
@@ -140,10 +140,10 @@
 REGISTER_DEVICES(device_table) = {
-	{"uart2", &uart_dev_ops, (void *)&uart2_data },
+	{"uart1", &uart_dev_ops, (void *)&uart1_data },
 };
 
 #ifdef CONFIG_DEBUG_ENABLE
 void debug_uart_init()
 {
-    uart_init(&uart2_data);
+    uart_init(&uart1_data);
 }
 #endif
