diff --git a/apps/common/ble/le_trans_data.c b/apps/common/ble/le_trans_data.c
--- a/apps/common/ble/le_trans_data.c
+++ b/apps/common/ble/le_trans_data.c
@@ -42,7 +42,7 @@ static u32 trans_recieve_test_count;
 #define TEST_AUDIO_DATA_UPLOAD       0 //测试文件上传
 
 
-#if 1
+#if 0
 extern void printf_buf(u8 *buf, u32 len);
 #define log_info          printf
 #define log_info_hexdump  printf_buf
diff --git a/apps/common/config/log_config/lib_btctrler_config.c b/apps/common/config/log_config/lib_btctrler_config.c
--- a/apps/common/config/log_config/lib_btctrler_config.c
+++ b/apps/common/config/log_config/lib_btctrler_config.c
@@ -276,7 +276,7 @@ const char log_tag_const_e_LL_TWS_SCAN AT(.LOG_TAG_CONST) = CONFIG_DEBUG_LIB(TRU
 const char log_tag_const_v_LL_S AT(.LOG_TAG_CONST) = CONFIG_DEBUG_LIB(FALSE);
-const char log_tag_const_i_LL_S AT(.LOG_TAG_CONST) = CONFIG_DEBUG_LIB(TRUE);
-const char log_tag_const_d_LL_S AT(.LOG_TAG_CONST) = CONFIG_DEBUG_LIB(TRUE);
-const char log_tag_const_w_LL_S AT(.LOG_TAG_CONST) = CONFIG_DEBUG_LIB(TRUE);
-const char log_tag_const_e_LL_S AT(.LOG_TAG_CONST) = CONFIG_DEBUG_LIB(TRUE);
+const char log_tag_const_i_LL_S AT(.LOG_TAG_CONST) = CONFIG_DEBUG_LIB(FALSE);
+const char log_tag_const_d_LL_S AT(.LOG_TAG_CONST) = CONFIG_DEBUG_LIB(FALSE);
+const char log_tag_const_w_LL_S AT(.LOG_TAG_CONST) = CONFIG_DEBUG_LIB(FALSE);
+const char log_tag_const_e_LL_S AT(.LOG_TAG_CONST) = CONFIG_DEBUG_LIB(FALSE);
 
 const char log_tag_const_v_LL_RL AT(.LOG_TAG_CONST) = CONFIG_DEBUG_LIB(FALSE);
