diff --git a/apps/common/config/log_config/lib_btctrler_config.c b/apps/common/config/log_config/lib_btctrler_config.c
--- a/apps/common/config/log_config/lib_btctrler_config.c
+++ b/apps/common/config/log_config/lib_btctrler_config.c
@@ -114,6 +114,6 @@ const int config_multilink_support_esco_enalbe = 0;
 #endif
 
-const int config_btctler_le_slave_multilink = 1;
+const int config_btctler_le_slave_multilink = 0;
 
 #if TRANS_MULTI_BLE_EN
 const int config_btctler_le_hw_nums = TRANS_MULTI_BLE_SLAVE_NUMS + TRANS_MULTI_BLE_MASTER_NUMS;
