diff --git a/apps/common/ble/le_trans_data.c b/apps/common/ble/le_trans_data.c
--- a/apps/common/ble/le_trans_data.c
+++ b/apps/common/ble/le_trans_data.c
@@ -102,6 +102,6 @@
 //---------------
 // 广播周期 (unit:0.625ms)
-#define ADV_INTERVAL_MIN          (160*5)
+#define ADV_INTERVAL_MIN          (32) /* 20 ms in 0.625 ms units */
 
 #define HOLD_LATENCY_CNT_MIN  (3)  //(0~0xffff)
 #define HOLD_LATENCY_CNT_MAX  (15) //(0~0xffff)
