diff --git a/apps/common/net/wifi_conf.c b/apps/common/net/wifi_conf.c
--- a/apps/common/net/wifi_conf.c
+++ b/apps/common/net/wifi_conf.c
@@ -728,7 +728,9 @@ static int wl_set_ssid(const char *ssid)
 
     strcpy(ssid_position, ssid);
     ssid_position[strlen(ssid)] = '\n';
-    memset(ssid_position + strlen(ssid) + 1, '#', 32 - strlen(ssid) - 1);
+    if (strlen(ssid) < 32) {
+        memset(ssid_position + strlen(ssid) + 1, '#', 32 - strlen(ssid) - 1);
+    }
 
     return 0;
 }
@@ -757,7 +759,9 @@ static int wl_set_passphrase(const char *passphrase)
 
         strcpy(wpa_passphrase_position, passphrase);
         wpa_passphrase_position[strlen(passphrase)] = '\n';
-        memset(wpa_passphrase_position + strlen(passphrase) + 1, '#', 63 - strlen(passphrase) - 1);
+        if (strlen(passphrase) < 63) {
+            memset(wpa_passphrase_position + strlen(passphrase) + 1, '#', 63 - strlen(passphrase) - 1);
+        }
 
         AUTH_MODE = "WPA2PSK";
         strcpy(AuthMode_position, AUTH_MODE);
